crates/forge/bin/cmd/script/build.rs

```diff
@@ -31,21 +33,23 @@ impl ScriptArgs {
         let (project, output) = self.get_project_and_output(script_config)?;
         let output = output.with_stripped_file_prefixes(project.root());
 
-        let mut sources: BTreeMap<u32, String> = BTreeMap::new();
+        let mut sources: ContractSources = HashMap::new();
```

Comment:
this is one reason why I dislike this so much

Reply:
Understood